twilio-sheet/README.md

```diff
@@ -0,0 +1,104 @@
+messages from a Google Sheet. It includes:
+
+1. A Google Sheet script that includes a custom menu to send text messages in
+   bulk via an integratation with Twilio.
+1. A basic web app (standalone script) that collects phone numbers from
+   external (optional to use).
+
+If do not wish to maintain a web app, you can link a Google Form to the Sheet
+and only collect one value from users called "phone number" in order to use the
+script _as is_.
+
+## Prerequisite
+
+- Access to creating a Google Sheet
+- Twilio text messaging account (contains a free trial)
+
+## Helpful resources
+
+- [Web apps on top of Google scripts](https://developers.google.com/apps-script/guides/web)
+- [Standalone script (unbound) App Script](https://developers.google.com/apps-script/guides/standalone)
+- [Container bound script (tied to Sheet) Apps Script](https://developers.google.com/apps-script/guides/bound)
+- [Twilio's SMS instructions](https://www.twilio.com/blog/2016/02/send-sms-from-a-google-spreadsheet.html)
+- [Public Google Sheets Web App Tutorial](https://youtu.be/RRQvySxaCW0)
+
+## Try it
+
+1. Create a Twilio account (free trial comes with $15 to spend).
+Buy an SMS enabled phone number for $1(it’s within the trial’s free spend)
+Visit Twilio dashboard to locate API credentials
+
+    1. [Create a Twilio account](http://twilio.com/try-twilio?_ga=2.219940382.608987616.1576808306-1604173987.1564005873)
+       (free trial comes with $15 to spend).
+    1. [Buy an SMS enabled phone number for $1](https://www.twilio.com/user/account/phone-numbers/search)(it’s within the trial’s free spend)
+    1. Visit [Twilio dashboard](https://www.twilio.com/console)to locate API credentials
+
+1. Create the web app (an unbound script)
+    1. Visit Script.google.com > New script
+    1. Replace the existing code (if any) with [the code in the Code.gs tab](https://script.google.com/d/1lw89vpdMxVXRWWzGfqbnYyxYoZN9fhIZqb2wCdGyOTZrly9XQzqf1mV3/edit?usp=drive_web&folder=1Hs95J2PriOyFZOowx28O5xS-Bwb-iImx&splash=yes)
+        - Then save (File > Save)
+        - Create another file by clicking File > New > HTML file> call it
+          “Page.html” and paste the code from the [Page.html tab here.](https://script.google.com/d/1lw89vpdMxVXRWWzGfqbnYyxYoZN9fhIZqb2wCdGyOTZrly9XQzqf1mV3/edit?usp=drive_web&folder=1Hs95J2PriOyFZOowx28O5xS-Bwb-iImx&splash=yes)  
+        - Then save (File > Save)
+
+1. [Copy this Google Sheet](https://docs.google.com/spreadsheets/d/1A0YN5Ps2Iq-yTHvBPXwrz61HBXA0ZWgIML911ub6B3o/copy)
+    1. Copy the Sheet’s URL from the browser
+    1. Visit the web app’s script
+    1. Enter the URL of your spreadsheet at the top.
+
+1. Publish web application
+    1. Click Publish > Deploy as a web app > Deploy (leave all the default
+       options selected).
+    1. You will be given a URL, copy it and paste in a new tab in your browser.
+    1. This will bring up the web app that looks like this (I did not make it public so I am sharing the screenshot):
```

Comment:
It's not necessary to state that you didn't make it public. Showing the screenshot should be to hint people that it should look like that, so no need for this "step" altogether.